### ISSUE TITLE
Is project maintained ?

### ISSUE THREAD

**Issue by Alcamoru** [BenFradet/RiotSharp#725]
Hey, I'm new to C# language so I don't know if I'm making a mistake, but it seems that this project is no longer maintained. I would like to know if it's right to find another package.
Thanks!

**Comment by AoshiW** [comment]
> Is project maintained?

No, you should find another project (or you can ask for alternatives on [Discord](https://discord.com/invite/riotgamesdevrel)
